### PR TITLE
Use fallback route for 404s

### PR DESCRIPTION
**Pull request by AlexVanderbist**
For #437 

Are they really better tho? I feel like this is a bit unnecessary if we're not using anything special (sessions, cookies, auth, ...) in the 404 view/controller.

### CODE REVIEW

File: app/Http/Controllers/Front/NotFoundController.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Http\Controllers\Front;
+
+use App\Http\Controllers\Controller;
+
+class NotFoundController extends Controller
+{
+    public function __invoke()
+    {
+        return view('errors.404');
```
Comment (freekmurze) [inline app/Http/Controllers/Front/NotFoundController.php]: This is off. Looking at the code a view with back layout will be displayed at the front.

Lets create both a the `front.errors.404` and `back.errors.404` views so we can easily customise the front one in each project.

**Review by freekmurze — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Front/NotFoundController.php (hunk 1 above)
Comment (AlexVanderbist) [reply app/Http/Controllers/Front/NotFoundController.php]: The `errors.404` view actually did exactly that: 
`@include('errors.404-'.(request()->isBack() ? 'back' : 'front'))`

However I'll move the 404 views to the `front` and `back` directories and get rid of the `errors.404` view.

Comment (freekmurze) [reply app/Http/Controllers/Front/NotFoundController.php]: 👍

**Comment by sebastiandedeyne** [comment]
Can this be merged?

**Comment by AlexVanderbist** [comment]
Ready for me 👍

**Pull request merged**